packages/react-components/src/hooks/useECharts/useGroupableEchart.ts

```diff
@@ -10,10 +11,12 @@ import type { ECharts } from 'echarts';
  * @returns void
  */
 export const useGroupableEChart = (chartRef: React.MutableRefObject<ECharts | null>, groupId?: string) => {
+  const { group } = useViewport();
```

Comment:
Curious: what is this group ? 

previously this was dedicated to "group" charts, i.e have sync mode turned ON

Reply:
yeah, so we're still relying on this group as the source of truth since we have a mix of synchro-charts and echarts 

once we are solely using echarts we can stop relying on the useViewport group!